### ISSUE TITLE
bin/installUtility is not available [A feature definition cannot  be found...]

### ISSUE THREAD

**Issue by saschaschefenacker** [OpenLiberty/open-liberty#363]
In liberty the command bin/installUtility was used to load the required features.
In openLiberty this command is not available - but the error message still tells me:

[ERROR   ] CWWKF0042E: A feature definition cannot  be found for the  openidconnectclient-1.0 feature.  Try running the command, bin/installUtility install openidconnectclient-1.0,  to install the feature. Alternatively, you can run the command, bin/installUtility install ssoServer,  to install all features that are referenced by this configuration.

Either the tool needs to be available and working
Or docu is required how to install features in openLiberty.

**Comment by ewhoch** [comment]
The messages should be updated to not reference the installUtility in this circumstance. The installUtility was used to install features from the WAS Liberty repository, in OpenLiberty the feature set in the packaged zip is definitive, although at the moment our nightly doesn't correctly have non-released features in it yet. Regardless, openidconnectclient-1.0 isn't in OpenLiberty.

**Comment by bmhm** [comment]
@ewhoch  thanks for the clarification.
Just being curious: Can I remove features, so I can keep my docker images small? Guacamole for example only needs Servlet, JSP and Websockets.

**Comment by ewhoch** [comment]
@bmhm yes, you can remove features you're not using, this is built in using a server minify which will repackage the server using only the features you declare in the server.xml.

This is done using the command:

"bin/server package serverName --include=minifiy"

**Comment by cbridgha** [comment]
PR: #1518

**Comment by cbridgha** [comment]
Closing - this was fixed by #1518
